zipkin-lens/src/components/App/Layout.tsx

```diff
@@ -63,20 +63,20 @@ const Layout: React.FC = ({ children }) => {
                 justifyContent="center"
                 alignItems="center"
               >
-                <Logo alt={i18n._(t`Zipkin`)} />
+                <Logo alt={_(msg`Zipkin`)} />
```

Comment:
https://lingui.dev/tutorials/react-patterns#element-attributes-and-string-only-translations